hardware/index.md

```diff
@@ -242,9 +241,9 @@ main computer.
 | Core variant: | [ARM Cortex-M0+](https://www.arm.com/products/processors/cortex-m/cortex-m0plus.php)
 | Flash ROM     | 128KB
 | RAM           | 16KB
-| Speed         | 16MHz
+| Speed         | [16Mhz (crystal)](https://github.com/ARMmbed/DAPLink/blob/master/source/hic_hal/freescale/kl26z/MKL26Z4/system_MKL26Z4.c#L69) 48MHz (max)
 | Debug capabilities | SWD
-| More Info | [here](/software/daplink-interface/)
+| More Info | [DAPLink](/software/daplink-interface/) [KL26 reference manual](http://cache.freescale.com/files/microcontrollers/doc/ref_manual/KL26P121M48SF4RM.pdf) [KL26Z data sheet](http://www.nxp.com/docs/pcn_attachments/16440_KL26P64M48SF5_Rev.4.pdf)
```

Comment:
Without some commas or something else between the links it can be hard to tell there are more than one.

Reply:
Ack. added commas